tldw_Server_API/app/core/Watchlists/output_enrichment_handler.py

```
@@ -0,0 +1,200 @@
+    from tldw_Server_API.app.services.outputs_service import (
+        _summarize_text_block,
+        classify_items_by_topic,
+        generate_briefing_summary,
+        generate_group_summaries,
+        group_items,
+        render_output_template,
+    )
+
+    try:
+        collections_db = _get_collections_db_for_user_id(user_id)
+    except Exception as exc:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)